recipes/fgmp/meta.yaml

```diff
@@ -0,0 +1,64 @@
+package:
+  name: fgmp
+  version: 1.0.1
+
+source:
+  url: https://github.com/stajichlab/FGMP/archive/v1.0.1.tar.gz
+  sha256: 18a95eacaa96ea24bfcd5b848355c4b1b6845ea1d1e86c014ffdf8927ecc6385
+
+build:
+  number: 0
+
+requirements:
+  build:
+    - {{ compiler('c') }}
```

Comment:
You are not compiling at all isn't it? Do you really need the compiler?

Reply:
That’s correct. Let’s remove